docs/rules/no-magic-number.md

```diff
@@ -0,0 +1,50 @@
+# Disallow Magic Numbers (no-magic-number)
+
+'Magic numbers' are numbers that occur multiple time in code without an explicit meaning.
+They should preferably be replaced by named constants.
+
+```js
+if (foo == null) {
+  bar();
+}
+```
```

Comment:
It's not. I used another doc as a template and failed to clean it up properly :/. 
Will it make travis' build fail again if I fix this in an another commit ?